nixos/modules/services/networking/jotta-cli.nix

```
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.user.services.jotta-cli;
+in {
+  options = {
+    user.services.jotta-cli = {
+
+      enable = mkEnableOption (mdDoc "Whether to enable the Jottacloud Command-line Tool.");
+
+      options = mkOption {
+        default = [ "stdoutlog" "datadir" "%h/.jottad/" ];
+        example = [ ];
+        type = with types; listOf str;
+        description = mdDoc "Command-line options passed to jottad.";
+      };
+    };
+  };
+  config = mkIf cfg.enable {
+    systemd.user.services.jottad = {
+
+      description = "Jottacloud Command-line Tool daemon";
+
+      serviceConfig = {
+        Type = "notify";
+        EnvironmentFile = "-%h/.config/jotta-cli/jotta-cli.env";
+        ExecStart = "${pkgs.jotta-cli}/bin/jottad ${concatStringsSep " " cfg.options}";
```

```suggestion
        ExecStart = "${lib.getExe cfg.package} ${concatStringsSep " " cfg.options}";
```
Please add a package option

---

Fixed, but with `getExe'` since the package does not have a single main application.